portfolio/src/main/webapp/script.js

```diff
@@ -42,4 +42,18 @@ async function getRandomQuoteUsingAsyncAwait() {
   const response = await fetch('/data');
   const quote = await response.text();
   document.getElementById('quote-container').innerText = quote;
+}
+
+async function loadTasks() {
+    let list = await fetch('/list-tasks').then(response => response.json());
```

Comment:
I don't think it's typical to mix await with promises like this. Stick to one or the other (i.e.):

```
let response = await fetch('/list-tasks')
let list = response.json()
```

Reply:
`let list = await response.json()` I think?